### DISCUSSION TITLE
Pasting into web

### DISCUSSION THREAD

**Discussion by ChildishGiant** [elementary/screenshot-tool#223] [Q&A] [answered]
I can't seem to paste copied screenshots into web (eg uploading to github or on anything like photopea) I imagine this might be an issue with web, not screenshot but wanted to ask here if anyone has had similar issues

**Comment by davidmhewitt** [comment, marked as answer]
It appears to be an issue with web as pasting an image from screenshot into a GitHub issue in Firefox appears to work.

**Reply by ChildishGiant** [reply]
Do we make patches to web for functionality or does that all happen upstream? Either way i might take a look at this and a few other nitpicks
